### PR TITLE
rspec category

### PR DESCRIPTION
**Pull request by tranghoangthu812102**
- Test controller + model category
Screenshot from 2021-08-31 22-12-01
Screenshot from 2021-08-31 22-12-12
Screenshot from 2021-08-31 22-12-30
Screenshot from 2021-08-31 22-12-36
Screenshot from 2021-08-31 22-12-47

### CODE REVIEW

**Comment by tranghoangthu812102** [comment]
ready

File: spec/controllers/categories_controller_spec.rb [hunk 1]
```diff
@@ -0,0 +1,218 @@
...
+  end
+
+  describe "DELETE #detroy" do
+    context "When user is admin" do
+      before do
+        log_in admin
+      end
+      let!(:category_count){Category.count}
+
+      before{delete :destroy, params: {id: category_1.id}}
```
Comment (ngocvt-0484) [inline spec/controllers/categories_controller_spec.rb:185]: gộp vs before dòng 180 dc k?

File: spec/factories/category.rb [hunk 2]
```diff
@@ -1,5 +1,5 @@
 FactoryBot.define do
   factory :category do
-    name { Faker::Name.name }
+    name { "hi" }
```
Comment (ngocvt-0484) [inline spec/factories/category.rb:3]: sao k dùng Faker ?

File: spec/controllers/categories_controller_spec.rb [hunk 3]
```diff
@@ -0,0 +1,218 @@
...
+
+      context "destroy success" do
+        it "create success" do
```
Comment (ngocvt-0484) [inline spec/controllers/categories_controller_spec.rb:188]: tên contexxt là destroy mà it lại create success là sao

**Comment by tranghoangthu812102** [comment]
ready

**Pull request merged**